packages/accounts/src/msca/plugins/types.ts

```diff
@@ -15,5 +15,5 @@ export type Plugin<TAbi extends Abi = Abi> = {
   getContract: <C extends Client>(
     client: C,
     address?: Address
-  ) => GetContractReturnType<TAbi, PublicClient, undefined, Address>;
+  ) => GetContractReturnType<TAbi, PublicClient, Address>;
```

Comment:
oh did this go from 4 to 3 generic types when upgrading viem?

Reply:
yup